pkg/server/backend_manager.go

```diff
@@ -85,7 +87,25 @@ type backend struct {
 func (b *backend) Send(p *client.Packet) error {
 	b.mu.Lock()
 	defer b.mu.Unlock()
-	return b.conn.Send(p)
+	errChan := make(chan error, 1)
+	go func() {
+		err := b.conn.Send(p)
+		errChan <- err
+		close(errChan)
+		if err != nil {
+			klog.ErrorS(err, "SendMsg has exited abnormally")
```

Comment:
I think we can remove this log line because the caller of Send (mainly from pkg/server/server.go) is wrapping or loggin the returned error already

Reply:
This would also simplify the code here to just:
```
go func() {
    errChan <- b.conn.Send(p)
    close(errChan)
}()

...
...
```

Reply:
Sure, good idea! Thank you!